lib/modules/dosomething/dosomething_api/resources/reportback_item_resource.inc

```diff
@@ -165,5 +165,5 @@ function _reportback_item_resource_kudos($fid, $user_id, $term_ids) {
   );
 
   // @TODO: temporarily returning back the header body data for testing.
```

Comment:
Ah nope. That whole targeted action is getting removed out of there the reportback items endpoint. 
I was planning on cleaning that up on next PR, but will update the TODO note to reflect the change. That endpoint still works, but will be removed. This was done prior to realizing I wouldn't be able to use a targeted action to complete all functionality for kudos.